app/services/bulk_api_import/fhir_importable.rb

```diff
@@ -0,0 +1,24 @@
+module BulkApiImport::FhirImportable
+  def import_user
+    ImportUser.find_or_create
+  end
+
+  def import
+    raise NotImplementedError
+  end
+
+  def timestamps
+    {
+      created_at: @resource.dig(:meta, :createdAt),
+      updated_at: @resource.dig(:meta, :lastUpdated)
+    }
+  end
+
+  def translate_id(id)
+    Digest::UUID.uuid_v5(Digest::UUID::DNS_NAMESPACE, id)
+  end
+
+  def translate_facility_id(id)
+    FacilityBusinessIdentifier.where(identifier_type: :external_org_facility_id, identifier: id).take.facility.id
```

Comment:
```suggestion
    FacilityBusinessIdentifier.find_by(identifier_type: :external_org_facility_id, identifier: id).facility.id
```

Reply:
This will also throw an uncaught exception here if the business identifier is not found, and the user will not get any feedback for this error. Maybe we should check in the controller that the facility business identifier exists for all the facility ids sent in the payload. Alternatively, maybe we can create dummy facilities for the organisation if the id they've sent doesn't exist in our system. Maybe we should discuss this a bit. 

Note although that this is not a blocker for the PR.

Reply:
Noted. There's another PR of the same stuff coming through, let's solidify an error handling strategy